requirements/requirements.txt

```diff
@@ -30,7 +29,6 @@ flask-cors==3.0.8
 flask-restful==0.3.7
 flask==1.0.3
 gevent==1.3.7
-gnureadline==6.3.8        # via ipython
```

Comment:
I believe this dependency is required for macos, @ulope ?

Reply:
Yes, same with `appnope` above. 

There are some issues about this in pip-tools:
- https://github.com/jazzband/pip-tools/issues/277
- https://github.com/jazzband/pip-tools/issues/563
- https://github.com/jazzband/pip-tools/issues/585

I think in this case we can get away with just doing nothing. Both are optional dependencies and won't really hurt if they're missing on mac.

Reply:
so, should we remove it or not?

Reply:
Yeah sorry, it's fine to remove both.